how-to/support-context-and-intents/public/apps.json

```diff
@@ -0,0 +1,386 @@
+    }],
+    "tags":[ "view", "fdc3", "interop", "openfin"]
+  },
+  {
+    "appId": "preload-tradingview-view",
+    "name": "preload-tradingview-view",
+    "title": "Trading View",
+    "description": "This is an example of a thirdparty site that is listening for contextual fdc3 messages and using the passed ticker to update itself through the use of preload scripts.",
+    "manifest": "http://localhost:8080/views/preload-tradingview-view.json",
+    "manifestType": "view",
+    "icons": [
+      { "src": "http://localhost:8080/images/icon-blue.png" }
+    ],
+    "contactEmail": "contact@example.com",
+    "supportEmail": "support@example.com",
+    "publisher": "OpenFin",
+    "intents": [],
+    "images": [],
+    "tags":["view", "fdc3", "preload"]
+  },
+  {
+    "appId": "preload-google-view",
+    "name": "preload-google-view",
+    "title": "Google View",
+    "description": "This is an example of a thirdparty site that is listening for contextual fdc3 messages and using the passed ticker to update itself through the use of preload scripts.",
+    "manifest": "http://localhost:8080/views/preload-google-view.json",
+    "manifestType": "view",
+    "icons": [
+      { "src": "http://localhost:8080/images/icon-blue.png" }
+    ],
+    "contactEmail": "contact@example.com",
+    "supportEmail": "support@example.com",
+    "publisher": "OpenFin",
+    "intents": [],
+    "images": [],
+    "tags":["view", "fdc3", "preload"]
+  },
+  {
+    "appId": "expero-company-news",
+    "name": "expero-company-news",
+    "title": "Gateway - Company News",
+    "manifest": "https://openfin-iex.experolabs.com/openfin/manifests/company-news.json",
+    "manifestType": "view",
+    "icons": [
+      {
+        "src": "https://openfin-iex.experolabs.com/favicon.ico"
+      }
+    ],
+    "contactEmail": "contact@example.com",
+    "supportEmail": "support@example.com",
+    "publisher": "Expero",
+    "intents": [],
+    "images": [{
+      "src":"http://localhost:8080/images/previews/expero-news-view.png"
+    }],
+    "tags":["expero", "view", "interop"]
+  },
+  {
+    "appId": "openfin-intent-page",
+    "name": "openfin-intent-page",
+    "title": "Receive Intent Page",
+    "manifestType": "snapshot",
+    "description": "Shows an example of a snapshot that is a page that has two views that can act as an intent target. This entry has a manifest type of 'snapshot'.",
+    "manifest": "http://localhost:8080/snapshot-intent-single-instance.json",
+    "icons": [
+      { "src": "http://localhost:8080/images/icon-blue.png" }
+    ],
+    "contactEmail": "contact@example.com",
+    "supportEmail": "support@example.com",
+    "publisher": "OpenFin",
+    "intents": [
+      {
+        "name": "ShowInstrument",
+        "displayName": "Show Instrument",
+        "contexts": [
+          "fdc3.instrument"
+        ],
+        "customConfig": {}},
+      {
+      "name": "ShowInstrumentForPage",
+      "displayName": "Show Instrument For Page",
```

Comment:
Could rename this to "Show Instrument (pages only)" for more clarity.

Reply:
updated